qlik-oss/sn-table
# [FIX#216]: fix for proper `aria-*` attributes in table sorting

This PR will add `aria-sort` and `aria-label` attributes for table head cells while sorting. 
currently it has been tested with a chrome extension ( this one: https://chrome.google.com/webstore/detail/empty-title/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en-GB ) and also with Mac OS's default voice over feature. 

Unit Tests:
added 2 tests for: 
1. check to see if `aria-sort` property has been rendered properly 
2. check to see if `aria-label` property has been rendered properly
